src/wallet/rpcwallet.cpp

```diff
@@ -1447,7 +1447,7 @@ static UniValue listsinceblock(const JSONRPCRequest& request)
                 "Additionally, if include_removed is set, transactions affecting the wallet which were removed are returned in the \"removed\" array.\n",
                 {
                     {"blockhash", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "If set, the block hash to list transactions since, otherwise list all transactions."},
-                    {"target_confirmations", RPCArg::Type::NUM, /* default */ "1", "Return the nth block hash from the main chain. e.g. 1 would mean the best block hash. Note: this is not used as a filter, but only affects [lastblock] in the return value"},
+                    {"target_confirmations", RPCArg::Type::NUM, /* default */ "1", "Return the nth block hash from the main chain or the last block hash in case the nth block does not exist. e.g. 1 would mean the best block hash. Note: this is not used as a filter, but only affects [lastblock] in the return value"},
```

Comment:
In commit "Return genesis hash" (c0e57fe0d399235f3f2612c4972c597b34107509)

I think this description is more confusing than before and there's no need to mention the special case here, since it's explained in lastblock documentation below. If you think it should be mentioned both places, maybe put the inserted text after "e.g. 1 would mean the best block hash" instead of before to avoid jumping topics.